Libraries/Utilities/ImagePickerIOS.js

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
I think `ImagePickerIOS.js` should be `ImagePickerIOS.ios.js` (as referenced in extractDocs).